src/formatting/html.ts

```diff
@@ -20,15 +21,31 @@ const outcomeIcons: { [key in TestOutcome]: string } = {
 export const formatTitleHtml = (title: string): string =>
   wrap(title, { tag: 'h1', attributes: { id: getSectionLink(title) } });
 
-export const formatResultHtml = (result: IResult): string => {
+export const formatResultHtml = (result: IResult, onlyShowFailedTests:boolean, showTestOutput:boolean): string => {
   let html = wrap('Tests', 'h3');
 
   html += formatTable(
     [{ name: '✔️ Passed' }, { name: '❌ Failed' }, { name: '⚠️ Skipped' }, { name: '⏱️ Time' }],
     [[`${result.passed}`, `${result.failed}`, `${result.skipped}`, formatElapsedTime(result.elapsed)]]
   );
 
-  html += result.suits.map(formatTestSuit).join('');
+  let filteredSuits = sort(result.suits).asc([s=> s.tests.filter(t=> t.outcome === 'Failed').length > 0 ? 0 : 1, s => s.name]);
```

Comment:
this can be slightly simplified to 
`let filteredSuits = sort(result.suits).asc([s => s.success ? 1 : 0, s => s.name]);`

Reply:
This was intentional. I first had the code like you commented above, however when there are skipped tests it didn't work as expected and these would be at the top when there were successes mixed in with the suite. Which is why I had to filter into the actual tests.